packages/gatsby/content/advanced/lexicon.md

```diff
@@ -67,6 +67,14 @@ See also: the [`Linker` interface](https://github.com/yarnpkg/berry/blob/master/
 
 See also: the [`Installer` interface](https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-core/sources/Installer.ts#L18)
 
+### Local Cache
+
+The local cache is a way to protect against registries going down by keeping a cache of your packages within your very project (often checked-in within the repository). While not always practical (it causes the repository size to grow, although we have ways to mitigate it significantly), it presents various interesting properties:
+
+- It doesn't require additional infrastructure, such as a [Verdaccio proxy](https://verdaccio.org/)
+- The install fetch step is as fast as it can be, with no data transfer at all
+- It lets you reach [zero-installs](https://yarnpkg.com/features/zero-installs) if you also use the PnP linker
+
```

Comment:
Another use case for the local cache, without committing it, is you can use it to avoid dealing with credentials in your Dockerfile

Reply:
Another use case unrelated to offline mode:  github actions cache: https://github.com/actions/setup-node/issues/325.